charts/manager/alerts/operator.alerts

```diff
@@ -28,6 +28,15 @@ groups:
       for: 15m
       labels:
         severity: warning
+    - alert: GreenhousePluginHelmChartTestFailures
+      expr: |
+        sum by(plugin, cluster, namespace)(rate(greenhouse_plugin_chart_test_runs_total{result="Error"}[30m])) > 0
```

Comment:
The range of the rate should be less than the `for` condition. This would fire as long as one test in the last 30m failed.

Reply:
thanks a lot! fixed in 3869b3f3d7de8beef50c6c10575a12d1d5272429